### ISSUE TITLE
google spreadsheet loading..

### ISSUE THREAD

**Issue by buildvoc10** [dai-shi/excalidraw-gallery#4]
when adding https://docs.google.com/spreadsheets/d/17m0x0X3b0OGa_3vvSbwM4q5l68urXu-bfpf6dNWSNI0/edit#gid=0
receive the following error 
Uncaught SyntaxError: Unexpected token 'cons
![Pasted File at October 25, 2021 1_35 PM (1)](https://user-images.githubusercontent.com/76884997/138732698-7ccaa61e-36ea-43b8-ad6c-4119bf872229.png)
t'

**Comment by dai-shi** [comment]
Can you try this using Chrome?

https://dai-shi.github.io/excalidraw-gallery/#spreadsheetId=17m0x0X3b0OGa_3vvSbwM4q5l68urXu-bfpf6dNWSNI0&gid=0

**Comment by buildvoc10** [comment]
Daishi thanks for the link still receiving the same error Uncaught SyntaxError: Unexpected token 'const' main.js:42 (am running this app on my own server)

**Comment by dai-shi** [comment]
"Unexpected token 'const'" sounds like it's not a modern browser? Can you try with Chrome?

Please let me know if the [direct link](https://dai-shi.github.io/excalidraw-gallery/#spreadsheetId=17m0x0X3b0OGa_3vvSbwM4q5l68urXu-bfpf6dNWSNI0&gid=0) works or not.

**Comment by buildvoc10** [comment]
Yes this direct link works, want to get my own gallery of fire safety diagrams

**Comment by dai-shi** [comment]
I guess you need to obtain your own apiKey in such a case. But, I don't remember exactly. Please learn from Sheets API docs.
